benchmarks/forms.py

```diff
@@ -7,49 +7,52 @@
 
 
 class SignupForm(UserCreationForm):
-	email = forms.EmailField(max_length=200, help_text='Required')
+    email = forms.EmailField(max_length=200, help_text='Required')
 
-	def save(self, commit=True):
-		user = super(SignupForm, self).save(commit=False)
-		# Uses the email from this custom form password1 from the UserCreation Form it extends from.
-		user.email = self.cleaned_data['email']
-		user.set_password(self.cleaned_data["password1"])
-		if commit:
-			user.save()
-			return user
+    def save(self, commit=True):
+        user = super(SignupForm, self).save(commit=False)
+        # Uses the email from this custom form password1 from the UserCreation Form it extends from.
+        user.email = self.cleaned_data['email']
+        user.set_password(self.cleaned_data["password1"])
+        if commit:
+            user.save()
+            return user
 
-	class Meta:
-		# Uses the password1 and password2 fields from the UserCreation Form
-		model = User
-		fields = ('email',)
+    class Meta:
+        # Uses the password1 and password2 fields from the UserCreation Form
+        model = User
+        fields = ('email',)
 
 
 class LoginForm(AuthenticationForm):
-	class Meta:
-		model = User
-		fields = ('email', 'password')
+    class Meta:
+        model = User
+        fields = ('email', 'password')
 
 
 class UploadPlaceHolder(forms.Form):
-	zip_file = forms.FileField(help_text='Required')
-	config_file = forms.FileField(help_text='Required')
+    zip_file = forms.FileField(help_text='Required')
+    config_file = forms.FileField(help_text='Required')
 
 
 class UploadFileForm(forms.Form):
-	name = forms.CharField(max_length=200, help_text='Required')
-	model_type = forms.ChoiceField(choices=[
-		("BaseModel", "BaseModel - to submit a standard machine learning model"),
-		("BrainModel", "BrainModel - to change brain-transformation, e.g. layer-mapping, visual degrees etc.")])
-	zip_file = forms.FileField(help_text='Required')
+    # name = forms.CharField(max_length=200, help_text='Required')
+    model_type = forms.ChoiceField(choices=[
+        ("BaseModel", "Base model - to submit a standard machine learning model"),
+        ("BrainModel", "Brain model - to change brain-transformation, e.g. layer-mapping, visual degrees etc.")])
+    public = forms.BooleanField(label='Make model scores public:', required=False,
+        help_text='Check if you want the results of your submitted models included in the public ranking.')
```

Comment:
I would add "You can also change this after your model has been scored." (in the user profile, there's a checkbox next to each model to make it public or not)